lib/customer.rb

```diff
@@ -0,0 +1,62 @@
+require_relative 'takeaway'
+
+class Customer
+
+  attr_accessor :order_basket, :order_total, :total_items, :takeaway
+
+  def initialize(klass = Takeaway)
+    @order_basket = {}
+    @order_total = 0
+    @total_items = 0
+    @takeaway = klass.new
+  end
+
+  def view_menu
+    @takeaway.menu.each { |key, value| p "#{key} - #{value}"}
+  end
+
+  def select_item(item, quantity)
+    fail 'Sorry, that item is not on the menu.' unless @takeaway.menu.has_key?(item)
+    update_order_basket(item, quantity)
+    update_order_total(item, quantity)
+    update_total_items(quantity)
+  end
+
+  def receipt
+     "Your order has " + total_items.to_s + " items with a total of £" +
+    order_total.to_s + '0'
```

Comment:
Style/MultilineOperationIndentation: Use 2 (not -1) spaces for indenting an expression spanning multiple lines.